docs/UserGuide.md

```diff
@@ -727,6 +727,7 @@ Each credit card tier is visually distinguished in the UI: Gold is marked with a
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
 2. **If you minimize the Help Window** and then run the `help` command (or use the `Help` menu, or the keyboard shortcut `F1`) again, the original Help Window will remain minimized, and no new Help Window will appear. The remedy is to manually restore the minimized Help Window.
+3. **When inputting names that use `/`**, this can cause an `invalid command` error to be shown or even unintended attributes for the client added. Avoid using `/`, and spell out the names in full, for cases like 'Ramesh s/o Ravichandran', change it to 'Ramesh Son Of Ravichandran' 
```

Comment:
I still having concern if banking agent should record the name as per officials', where s/o is allowed in official/gov document. So im thinking should we at least show on the list name with s/o (but then when user gg to search with cli it still needs to search by Son Of not s/o). Might have better way to mitigate/or defend our point about that but We can discuss about this more later

Reply:
I came up with some roughly draft pseudocode on the checking logic for s/o and d/o appeared under n/ flag in add and edit command, can consider this change if we have enough time in this iteration

```
    static final List<String> VALID_COMMANDS = ......;
    static final List<String> NAME_SUFFIXES = Arrays.asList("s/o", "d/o");
    
    public List<Command> parse(String input) {
        List<Command> commands = new ArrayList<>();
        int currentPos = 0;
        
        while (currentPos < input.length()) {
            String nextCommand = findNextCommand(input, currentPos);
            if (nextCommand == null) break;
            
            int commandStart = input.indexOf(nextCommand, currentPos);
            
            if (nextCommand.equals("n/")) {
                int nextRealCommand = findNextRealCommand(input, commandStart + nextCommand.length());
                String content = nextRealCommand != -1 
                    ? input.substring(commandStart + nextCommand.length(), nextRealCommand).trim()
                    : input.substring(commandStart + nextCommand.length()).trim();
                commands.add(new NameCommand(content));
                currentPos = nextRealCommand != -1 ? nextRealCommand : input.length();
            } else {
                int contentEnd = findContentEnd(input, commandStart + nextCommand.length());
                String content = input.substring(commandStart + nextCommand.length(), contentEnd).trim();
                commands.add(createCommand(nextCommand, content));
                currentPos = contentEnd;
            }
        }
        return commands;
    }
    
    private int findNextRealCommand(String input, int startPos) {
        int pos = startPos;
        while (pos < input.length()) {
            // find next slash
            pos = input.indexOf("/", pos);
            if (pos == -1) return -1;
            
            // check if it's part of the special name suffix
            boolean isNameSuffix = false;
            for (String suffix : NAME_SUFFIXES) {
                // check the slash if it's for s/o or d/o or not
                if (pos >= 1 && pos + 1 < input.length() && 
                    suffix.equals(input.substring(pos - 1, pos + 2))) {
                    isNameSuffix = true;
                    break;
                }
            }
            
            if (isNameSuffix) {
                pos += 2;
                continue;
            }
            
            // found a real command prefix
            if (pos >= 1 && VALID_COMMANDS.contains(input.substring(pos - 1, pos + 1))) {
                return pos - 1;
            }
            pos++;
        }
        return -1;
    }

```